ci-operator/config/stolostron/multicluster-observability-operator/stolostron-multicluster-observability-operator-main.yaml

```diff
@@ -171,12 +171,12 @@ tests:
     dependencies:
       COMPONENT_IMAGE_REF: multicluster-observability-operator
     env:
-      ACM_RELEASE_VERSION: release-2.9
+      ACM_RELEASE_VERSION: release-2.10
```

Comment:
Just an FYI, because there aren't any images available yet for `release-2.10`, this will cause the E2E test job to fail. If you plan to go this route, you would need to verify that the tests can run properly locally or possibly skip this job when creating a PR.

The other option is to stuck with `release-2.9` until the `release-2.10` images are available.